### PR TITLE
Replace hardcoded strings

### LINKED ISSUE #1612: Remove hardcoded strings to enable localization
Lots and lots of hardcoded strings; we should move these out to enable localization.

### PR DESCRIPTION
**Pull request by bwateratmsft**
Fixes #1612

### CODE REVIEW

**Comment by bwateratmsft** [comment]
@ravipal @haniamr @philliphoff @karolz-ms I hope you guys enjoy code-reviewing these mindless, repetitive changes as much as I enjoyed making them. 🤣

File: package.nls.json [hunk 1]
```diff
@@ -0,0 +1,246 @@
+{
+    "vscode-docker.description": "Makes it easy to create, manage, and debug containerized applications.",
```
Comment (bwateratmsft) [inline package.nls.json]: Need to validate that there are no duplicate IDs, here and in the source files themselves.

Comment (philliphoff) [reply package.nls.json]: Yeah, there is a definite need for a NLS extension to help with these sorts of editing and validation tasks; it's still very much a mysterious process to me.  Maybe if we can find a way to tie that to containers, management will let us work on one.  :-)

**Comment by karolz-ms** [comment]
😲 ☺️

File: src/debugging/coreclr/debugSessionManager.ts [hunk 2]
```diff
@@ -28,7 +29,7 @@ export class DockerDebugSessionManager implements DebugSessionManager, vscode.Di
                 () => {
                     this.dockerManager
                         .cleanupAfterLaunch()
-                        .catch(reason => console.log(`Unable to clean up Docker images after launch: ${reason}`));
+                        .catch(reason => console.log(localize('vscode-docker.debug.coreclr.cleanupFailed', 'Unable to clean up Docker images after launch: {0}', reason as string)));
```
Comment (philliphoff) [inline src/debugging/coreclr/debugSessionManager.ts]: Are we sure this is a string?

File: package.nls.json (hunk 1 above)
Comment (bwateratmsft) [reply package.nls.json]: Validated and found two duplicate IDs, now fixed.

**Pull request merged**